tensorflow/core/kernels/conv_ops_benchmark_test.cc

```diff
@@ -346,11 +347,11 @@ static Graph* FusedConv2DWithBatchNorm(
     test::Benchmark(#type, Conv2D<float>(N, H, W, C, FW, FH, FC).graph, \
                     /*old_benchmark_api=*/false)                        \
         .Run(state);                                                    \
-    BM_SET_INFO(N, H, W, C, type, LABEL, Conv2D);                       \
+    BM_SET_INFO(N, H, W, C, FW, FH, FC, type, LABEL, Conv2D);           \
   }                                                                     \
   BENCHMARK(BM_NAME(BM_Conv2D, type, N, H, W, C, FW, FH, FC))           \
       ->Arg(/*unused arg*/ 1)                                           \
-      ->MeasureProcessCPUTime();
+      ->MeasureProcessCPUTime()->UseRealTime();
```

Comment:
I don't love the idea of using different settings internally vs externally.

It sounds like the main issue is in the minimum benchmark time, but we can increase that via command-line parameter when running the benchmark.

Reply:
Thanks @penpornk and @cantonios. As suggested I removed this because I can see now that I am able to increase iteration count by passing from command line for example `--benchmark_min_time`